internal/dataplane/parser/translate_httproute.go

```diff
@@ -393,17 +425,26 @@ func generateKongstateHTTPRoute(routeName string, ingressObjectInfo util.K8sObje
 }
 
 // generatePluginsFromHTTPRouteFilters  converts HTTPRouteFilter into Kong filters.
```

Comment:
Nit: `into Kong filters` -> `into Kong plugins`?

Reply:
Can we also add to this comment the already existing `tags` and just added `path`?

Reply:
a568d9ccf72179b208b18877e12f5bde3f21d555 hopefully explains why the tags are the tags that they are.

That was not a good sentence. The comment sentence is hopefully better.